### PR TITLE
Added Home Page for My Brain Tube App

### PR DESCRIPTION
**Pull request by kiranpratyush**
This PR includes the screens and some functionality for the Homepage
1. Code still not fully refactored somewhere is been repeated. Please give your feedback
2. Logo is still of youtube but please ignore it. It will be changed

https://user-images.githubusercontent.com/77445957/161369940-d53a11d2-3c6e-44f1-a0b5-addd05afaf9b.mp4

[livelink](https://spiffy-fenglisu-8e9c7c.netlify.app/)

### CODE REVIEW

File: src/Components/Header/Header.css [hunk 1]
```diff
@@ -0,0 +1,54 @@
+.header{
+    display:flex;
+    align-items: center;
+    justify-content: space-between;
+    padding:20px;
```
Comment (Sharmaryan) [inline src/Components/Header/Header.css:5]: you can use rem here

File: src/Components/VideoRecommender/RecommendVideo.js [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+  useEffect(() => {
+    setLoading(true);
+    getVideoData().then((data) => {
+      setLoading(false);
+      setData([...data]);
+    });
+  }, []);
+
+  return (
+    // title, timeStamp, views, profileImage, videoImage,channel
```
Comment (Sharmaryan) [inline src/Components/VideoRecommender/RecommendVideo.js:18]: can remove this line

File: src/Pages/HomePage/HomePage.js [hunk 3]
```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import { SideBar } from '../../Components/SideBar/SideBar';
+import { Header } from '../../Components/Header/Header';
```
Comment (Sharmaryan) [inline src/Pages/HomePage/HomePage.js:3]: import these components in a separate file and then import from there , this way readability will be improved

**Review by Sharmaryan — APPROVED** [review]

File: src/Components/Header/Header.js [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+  function handleNavigate()
+  {
+    if(!state.user){
+      navigate("/auth/login")
+    }
+    else
+    {
+      dispatch({type:"SIGN_OUT"})
+    }
+  }
```
Comment (gopalbbharadva) [inline src/Components/Header/Header.js:20]: Suggestion : You can use ternary here.

**Comment by Sharmaryan** [comment]
UI looks nice, same like youtube, great work, I suggested small changes

**Review by gopalbbharadva — APPROVED** [review]
Ui looks good, Good to go.

File: src/Components/VideoRecommender/RecommendVideo.js (hunk 2 above)
Comment (kiranpratyush) [reply src/Components/VideoRecommender/RecommendVideo.js]: Done

File: src/Pages/HomePage/HomePage.js (hunk 3 above)
Comment (kiranpratyush) [reply src/Pages/HomePage/HomePage.js:3]: Barel Export will be done later! Thank you for suggesting

File: src/Components/Header/Header.js (hunk 4 above)
Comment (kiranpratyush) [reply src/Components/Header/Header.js]: Thank you for the suggestion. Implemented

**Pull request merged**